charts/shoot-core/charts/calico/templates/calico.yaml

```diff
@@ -388,6 +388,7 @@ metadata:
   name: felixconfigurations.crd.projectcalico.org
   labels:
     addonmanager.kubernetes.io/mode: Reconcile
+    garden.sapcloud.io/role: system-component
```

Comment:
@adracus, @rfranzke what do you think about `gardener.cloud/role`?

Reply:
Generally yes, but doing this consistently would roll all the pods in the kube-system namespace. Should we try to combine this change with e.g. a version/image update (where the pods needs to be rolled anyways)?